src/main/java/io/zipcoder/interfaces/Instructors.java

```diff
@@ -0,0 +1,60 @@
+package io.zipcoder.interfaces;
+
+//Part 7.1 - Create Students singleton
+//
+//        Note: The creation of this class will demonstrate an implementation of singleton design pattern.
+//        Create a Students class.
+//        The class should be an unextendable subclass of the People class.
+//        The class should statically instantiate a final field named INSTANCE of type Students.
+//        The class should define a private nullary constructor which populates the INSTANCE field with respective
+//        io.zipcoder.interfaces.Student representations of your colleagues.
+//        Each student should have a relatively unique id field.
+//        The class should define a getInstance method which returns the INSTANCE field.
+
+
+public class Instructors extends People<Instructor>{
+
```

Comment:
You can eliminate your static initializer and clean up some of the initialization by using a structure which resembles the block below.
```java
private static final Instructors INSTANCE = new Instructors();
private Instructors() {
    String[] names = {"Leon", "Tariq", "Dolio", "Iyasu", "Froilan"};
    for(String name : names) {
        super.addPerson(new Instructor(name));
    }
}
```